### DISCUSSION TITLE
different sized input images

### DISCUSSION THREAD

**Discussion by matanPercepto** [zju3dv/LoFTR#204] [Q&A]
Hi,
I ran into a problem while trying to run loftr matcher:
I was trying to run matcher on 2 images with different sizes (I resized them to be a size divisible by 8, but kept the aspect ratio, that is different in each image), but receive error in the positional encoding stage that the tensors must be same size for concatenation.
I didn't see any requirement regarding this in the documentation, but is it the actual issue? both images must be the same size?
Thanks!

**Comment by SARAH-HADDAD** [comment]
did you find a solution ?
